test/Microsoft.Extensions.SecretManager.Tests/ConfigurationExtensionTests.cs

```diff
@@ -44,7 +44,7 @@ public void AddUserSecrets_With_An_Existing_Secret_File()
             var projectPath = UserSecretHelper.GetTempSecretProject(out userSecretsId);
```

Comment:
If there was anything actually wrong with the tests that'd be one thing. But just shuffling files around...